Makefile

```diff
@@ -9,6 +9,9 @@ images:
 install:
 	./hack/install.sh
 
+install-operator:
```

Comment:
`install-operator-only` ?

Reply:
None of the other targets (like `install-serving`) specify the "only" bit though they all mean "xxx only", so I'd personally keep that for consistency. This has been broken out to https://github.com/openshift-knative/serverless-operator/pull/923 anyway since it's not directly related here.

Reply:
yeah. It's perhaps just for us anyways - but I was wondering if "install operator" might be misunderstood.

Just a nit comment. I am fine w/ keeping it, though, just was wondering

Reply:
> they all mean "xxx only"

that actually is a good point, Markus